movie_backend/movie_api/tests.py

```diff
@@ -0,0 +1,68 @@
+from django.test import TestCase
+from rest_framework.test import APITestCase, APIClient
+from rest_framework import status
+from .models import Movie
+import json
+from django.urls import reverse
+# from rest_framework.test import APIRequestFactory
+# Create your tests here.
+
+
+class MovieTest(APITestCase):
+
+    def test_post(self):
+        data = {
```

Comment:
Move test data into some kind of mock file to have test data separated from test codes